docs/upgrade-guides/core/3-20-to-3-21.mdx

```diff
@@ -304,4 +304,9 @@ More details can be found [`here`](developer/price-freeze-period.mdx)
 
 - Draft orders: the base prices are frozen for 24 hours by default. This duration can be modified using the `Channel.draftOrderLinePriceFreezePeriod` setting.
 - Unconfirmed orders: the base prices remain frozen indefinitely, and this setting is not configurable.
-- Other orders: remain unchanged and cannot be recalculated.
\ No newline at end of file
+- Other orders: remain unchanged and cannot be recalculated.
+
+
+## `change_user_address` method removed from Plugin Manager
+
+The change_user_address method has been removed from the Plugin Manager and is no longer available.
```

Comment:
this is confusing because its purely technical change and it's on the same "level" as functional changes

we can wrap it with some section like "implementation changes" or "plugin changes"

Reply:
There are some technical changes mentioned here already, e.g.  [## Deprecated `WebhookPlugin` and webhook-related plugin hooks](https://github.com/saleor/saleor-docs/pull/1508/files#diff-56e89c934a867e65c4daef54fbc03a45fb8b41d208b9a97ff5f728393bce2433R29), but it has a note that it affects only open-source users. We could group both parts under one section e.g. "Plugin changes", and keep the note that it only affects OSS.